projects/github-actions/repo-gardening/src/tasks/update-board/index.js

```diff
@@ -212,6 +307,66 @@ async function setPriorityField( octokit, projectInfo, projectItemId, priorityTe
 	return newProjectItemId; // New Project item ID (what we just edited). String.
 }
 
+/**
+ * Update the "Status" field in our project board.
+ *
+ * @param {GitHub} octokit       - Initialized Octokit REST client.
+ * @param {object} projectInfo   - Info about our project board.
+ * @param {string} projectItemId - The ID of the project item.
+ * @param {string} statusText    - Status of our issue (must match an existing column in the project board).
+ * @returns {Promise<string>} - The new project item id.
+ */
+async function setStatusField( octokit, projectInfo, projectItemId, statusText ) {
+	const {
+		projectNodeId, // Project board node ID.
+		status: {
+			id: statusFieldId, // ID of the status field.
+			options,
+		},
+	} = projectInfo;
+
+	// Find the ID of the status option that matches our issue status.
+	const statusOptionId = options.find( option => option.name === statusText ).id;
```

Comment:
**suggestion (non-blocking):** We'll get a `TypeError` when there's no matching status option in the array. We could use `options.find( option => option.name === statusText )?.id` 

In that way `if (!statusOptionId)` will catch this condition.

Reply:
Good point. I just pushed 5e39ae90e7c5e843838bbe6d4d28f5d2fdebb700 to address that.